ocaml-lsp-server/src/diagnostics.mli

```diff
@@ -0,0 +1,16 @@
+open Import
+
+type t
+
+val create : (PublishDiagnosticsParams.t -> unit Fiber.t) -> t
+
+val send : t -> unit Fiber.t
+
+val set :
+     t
+  -> [ `Dune of int * Uri.t * Diagnostic.t
+     | `Merlin of Uri.t * Diagnostic.t list
+     ]
+  -> unit
+
+val remove : t -> [ `All_dune | `Dune of int | `Merlin of Uri.t ] -> unit
```

Comment:
Why is the variant `Dune` taking an `int`? You're passing a `Uri.t` in `set` as well, so this is a bit confusing reading the interface. Maybe some docstring if this is justified?

Reply:
You answered that above.